kythe/proto/analysis.proto

```diff
@@ -188,25 +188,27 @@ message KzipInfo {
     // The count of data present in compilation units for a given corpus and
     // language.
     message CompilationUnits {
```

Comment:
Do we need this wrapper message anymore?

Reply:
I'd like to keep it so if we decide we want to record more info about the CU in the future we don't have to make a new map again

Reply:
But if you prefer I can get rid of it

Reply:
:+1: